modules/openxAnalyticsAdapter.js

```diff
@@ -954,10 +956,13 @@ function onBidResponse(bidResponse) {
 }
 
 function onBidTimeout(args) {
-  utils
-    ._map(args, value => value)
-    .forEach(({ auctionId, adUnitCode, bidId: requestId }) => {
-      auctionMap[auctionId].adUnitCodeToBidderRequestMap[adUnitCode][requestId].timedOut = true;
+  utils._each(args, ({auctionId, adUnitCode, bidId: requestId}) => {
+      try{
+        auctionMap[auctionId].adUnitCodeToBidderRequestMap[adUnitCode][requestId].timedOut = true;
+      } catch (e) {
+        utils.logError(`ERROR: oxAnalyticsAdapter: unable to track timeout for 
```

Comment:
This is absorbing the error, should we print it out?

Reply:
I found intermittent errors while testing, but was not able to debug the problem, nor get a printout of the error, so I left it here just in case.

Reply:
Can't we just add `e` as a 2nd argument? 

I'd hate to see this but not understand what the error is.

Reply:
The exception is that it can't find the requestId, which makes no sense because all requests have an id and i tracked them.

Reply:
k seems fine